Loop/Extensions/DeviceDataManager+SimpleBolusViewModelDelegate.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  DeviceDataManager+SimpleBolusViewModelDelegate.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 9/30/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import HealthKit
+import LoopCore
+import LoopKit
+
+extension DeviceDataManager: SimpleBolusViewModelDelegate {
+    func addGlucose(_ samples: [NewGlucoseSample], completion: @escaping (Error?) -> Void) {
+        loopManager.addGlucose(samples) { (result) in
+            switch result {
+            case .failure(let error):
+                completion(error)
+            case .success:
+                completion(nil)
+            }
+        }
+    }
+    
+    func addCarbEntry(_ carbEntry: NewCarbEntry, completion: @escaping (Error?) -> Void) {
+        loopManager.addCarbEntry(carbEntry) { (result) in
+            switch result {
+            case .failure(let error):
+                completion(error)
+            case .success:
+                completion(nil)
+            }
+        }
+    }
+    
+    func enactBolus(units: Double, at startDate: Date) {
+        enactBolus(units: units, at: startDate) { (_) in }
+    }
+    
+    func computeSimpleBolusRecommendation(carbs: HKQuantity?, glucose: HKQuantity?) -> HKQuantity? {
+        return loopManager.generateSimpleBolusRecommendation(carbs: carbs, glucose: glucose)
+    }
+    
+    var maximumBolus: Double {
+        return loopManager.settings.maximumBolus!
```

Comment:
Therapy settings are going to be enforced during launch. This is already the case in dev in DIY. Eventually we should have a TherapySettings like object that has non-optionals that the rest of the code can use. But for now, I think force-unwrapping settings is an ok stepping stone. We already have bad issues if you try to use different parts of Loop without settings, such as the settings interface itself, which does not support missing settings for a lot of cases.

Reply:
Ok ... for now ;).  I predict that we will see unexpected crashes, though, due to our policy around force-unwrapping.  In general, there are very few "acceptable" uses of it.  It usually is a code smell around other architectural deficiencies.  I agree that this PR may not be the time to address it, but I will continue to remark on it during reviews.  Thanks.

Reply:
Yeah, force unwrapping is definitely something to look at and see if the type system can be improved in some way. I think in this case, we do know it can be improved; it's just not the time to do it right now, I think.